src/apps/front-office/shop/pages/ComparePage/ComparePage.tsx

```diff
@@ -0,0 +1,101 @@
+import { trans } from "@mongez/localization";
+import Helmet from "@mongez/react-helmet";
+import { cn } from "@utils";
+import HorizonalTable from "src/apps/front-office/design-system/components/HorizonalTable";
+import Rating from "src/apps/front-office/design-system/components/Rating";
+import Breadcrumb from "src/apps/front-office/design-system/layouts/Breadcrumb";
+import URLS from "src/apps/front-office/utils/urls";
+import compareProducts, { StackStatus } from "../mocks/compare-products";
+
+export default function ComparePage() {
+  return (
+    <>
+      <Helmet title={trans("compare.title")} />
+      <main className="w-full text-secondary">
+        <div className="breadcrump relative">
+          <div className="absolute w-full border-b-2 border-b-gray-200 top-full"></div>
+          <div className="nav-links w-full">
+            <Breadcrumb
+              navItems={[
+                { name: "Pages" },
+                { name: "Compare Products", url: URLS.compare },
+              ]}
+              attributes={{
+                className: "justify-end items-end py-4 my-0",
+              }}
+            />
+          </div>
+        </div>
+
+        <section className="my-10">
+          <h2 className="text-4xl font-bold my-4">Compare Products</h2>
+          <p className="text-muted-foreground font-bold">
```

Comment:
This should be conditionally rendered because if there are compare products, this message will appear anyway